lib/python/qmk/info.py

```diff
@@ -472,6 +473,18 @@ def _extract_config_h(info_data):
     return info_data
 
 
+def _process_defaults(info_data):
+    """Proccess any additional defaults based on currently discovered infomation
+    """
+    defaults_map = json_load(Path('data/mappings/defaults.json'))
+    for default_type in defaults_map.keys():
+        thing_map = defaults_map[default_type]
+        if default_type in info_data:
+            for key, value in thing_map.get(info_data[default_type], {}).items():
+                info_data[key] = value
```

Comment:
```suggestion
                if key not in info_data:
                    info_data[key] = value
```

We probably don't want to overwrite the data from the `info.json` with the defaults.